ibc-testkit/src/relayer/error.rs

```diff
@@ -1,37 +1,27 @@
 use displaydoc::Display;
+
 use ibc::core::client::types::Height;
-use ibc::core::connection::types::error::ConnectionError;
 use ibc::core::handler::types::error::ContextError;
 use ibc::core::host::types::identifiers::ClientId;
 
 #[derive(Debug, Display)]
 pub enum RelayerError {
```

Comment:
I think we can eliminate the `RelayerError` entirely. This [`deliver`](https://github.com/cosmos/ibc-rs/blob/e721a5f749a7c625fd60b51e3bda19360b4dd126/ibc-testkit/src/context.rs#L483) method can just return `ContextError` since the error variant isn’t propagated anywhere. The other variants under `RelayerError` aren’t used either. They're only invoked within the `build_client_update_datagram` function as far as I see, but they’re neither propagated nor consumed later on.

Reply:
[ee365e3](https://github.com/cosmos/ibc-rs/pull/1321/commits/ee365e3c7753c27f95654dc3dc5fad0151c17fa1)